pkg/controller/nodeipam/ipam/multicidrset/multi_cidr_set.go

```diff
@@ -133,7 +133,7 @@ func NewMultiCIDRSet(cidrConfig *net.IPNet, perNodeHostBits int) (*MultiCIDRSet,
 	}
 
 	// Register MultiCIDRSet metrics.
-	registerCidrsetMetrics()
+	registerCidrSetMetrics()
```

Comment:
If the type is `MultiCIDRSet` then this function ought to be `registerCIDRSetMetrics`.
(In general, golang style is to never _partially_ uppercase acronyms within names; they are always either all-lowercase or all-uppercase.)

Reply:
OK, thank you, it has been modified to CIDR